bot/constants.py

```diff
@@ -88,6 +88,7 @@ class AdventOfCode:
     ignored_days = environ.get("AOC_IGNORED_DAYS", "").split(",")
     leaderboard_displayed_members = 10
     leaderboard_cache_expiry_seconds = 1800
+    max_dayandstar_results = 15
```

Comment:
Should we have variables that contain `dayandstar` be separated into `day_and_star`?

Reply:
I believe so, yes.